### ISSUE TITLE
DE43 does not parse international postcodes

### ISSUE THREAD

**Issue by margaretli27** [adelosa/cardutil#10]
The DE43 regex makes an assumption about the DE43_POSTCODE field, that it will always be 4-10 contiguous non-whitespace characters. However, not all postal codes particularly for non-US locations adhere to that. For instance, [British postal codes](https://ukpostcode.org/#:~:text=The%20postcode%20area%20and%20district,%2C%20and%20%22L1%208JQ%22.) can contain a space partway through, which causes the regex match to fail. And [Irish postal codes](http://www.geonames.org/postalcode-search.html?q=&country=IE) are three characters. I propose updating the default regex to be 
```
r"(?P<DE43_NAME>.+?) *\\(?P<DE43_ADDRESS>.+?) *\\(?P<DE43_SUBURB>.+?) *\\(?P<DE43_POSTCODE>.+?) *(?P<DE43_STATE>.{3})(?P<DE43_COUNTRY>.{3})$"}
```

For reference, the current regex is
```
r"(?P<DE43_NAME>.+?) *\\(?P<DE43_ADDRESS>.+?) *\\(?P<DE43_SUBURB>.+?) *\\(?P<DE43_POSTCODE>\S{4,10}) *(?P<DE43_STATE>.{3})(?P<DE43_COUNTRY>.{3})"
```

**Comment by adelosa** [comment]
Thanks for reporting this.. I have not looked deeply into international postcode formats so this seems like a reasonable approach.. I'll make some updates shortly and release an update.

**Comment by adelosa** [comment]
The final regex I have come up with is:
```
(?P<DE43_NAME>.+?) *\\(?P<DE43_ADDRESS>.+?) *\\(?P<DE43_SUBURB>.+?) *\\
(?P<DE43_POSTCODE>.{10})(?P<DE43_STATE>.{3})(?P<DE43_COUNTRY>\S{3})$
```

**Comment by margaretli27** [comment]
That looks reasonable to me!

**Comment by adelosa** [comment]
Great. I'll release an update with this change over the weekend.

**Comment by adelosa** [comment]
Updated in a1cbf0ee and released as v0.6.1.

**Comment by margaretli27** [comment]
Wonderful, thanks!
